plugins/item_tasks/web_client/templates/paginateTasksWidget.pug

```diff
@@ -3,6 +3,6 @@
 
 .list-group.g-task-list-container
   each task in tasks
-    a.list-group-item.g-execute-task-link(href=`#item_task/${task.id}/run`)
+    .list-group-item.g-execute-task-link(data-task-id=task.id)
```

Comment:
For reference or to lift ideas, https://github.com/girder/girder/pull/2004 also refactored the task list into a widget that's used from multiple views and changed the clickable list boxes to hyperlinks.